libraries/abstractions/platform/include/platform/transport_interface.h

```diff
@@ -1,4 +1,5 @@
 /*
```

Comment:
The `master` branch does not store a copy of the `transport_interface.h` file, instead just utilizes `coreMQTT` as the SSoT. Could you follow the same for this branch as well?

Reply:
Yea. Agree. I will do that in my next PR as that will need update to all the project files and retesting.